TourMate/TourMate/Frontend/Trip/Views/EditTripView/EditTripView.swift

```diff
@@ -27,11 +27,13 @@ struct EditTripView: View {
                         DatePicker(
                             "Start Date",
                             selection: $viewModel.trip.startDate,
+                            in: Date()...,
                             displayedComponents: [.date]
                         )
                         DatePicker(
                             "End Date",
                             selection: $viewModel.trip.endDate,
+                            in: viewModel.fromStartDate,
```

Comment:
We also need to enforce that end date > start date as well

Reply:
User won't be able to choose a end date that is < start date

Reply:
User won't be able to choose a end date that is < start date

Reply:
Okay my bad, didn't realize